pom.xml

```diff
@@ -38,6 +38,11 @@
         <tag>HEAD</tag>
     </scm>
     <dependencies>
+        <dependency>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-core</artifactId>
+            <version>1.18</version>
```

Comment:
Now that we're using Tika in multiple projects, it's probably a good idea to put it in the parent project. @janvanmansum @lindareijnhoudt please make issue!